README.md

```diff
@@ -10,52 +10,40 @@ A repository to migrate the pyOpenSci Jekyll site to Django with Wagtail CMS
 
 Before setting up the project locally, make sure you have the following installed:
 
-- **Python 3.8+** - [Download from python.org](https://www.python.org/downloads/)
+- **Python 3.12+** - [Download from python.org](https://www.python.org/downloads/)
+- **uv** - [Install uv](https://docs.astral.sh/uv/getting-started/installation/)
 - **Node.js** - [Download from nodejs.org](https://nodejs.org/) (for TailwindCSS)
```

Comment:
@Phinart98 isn't node.js installed via

`npm install` below? or is there another step i'm missing? thank you!